cognite_toolkit/_cdf_tk/loaders/_resource_loaders/transformation_loaders.py

```diff
@@ -430,6 +489,12 @@ def sensitive_strings(self, item: TransformationWrite) -> Iterable[str]:
         if item.destination_oidc_credentials:
             yield item.destination_oidc_credentials.client_secret
 
+        if item.external_id:
+            if read_credentials := self._authentication_by_id_operation.get((item.external_id, "read")):
+                yield read_credentials.client_secret
```

Comment:
couple potential issues here
- if external_id isn't set secrets should still be treated as sensitive
- is it possible that client_secret isn't present in credentials, would code still be fine?

Reply:
still need info here